### PR TITLE
Fix python

### PR DESCRIPTION
**Pull request by nlhepler**
fixes the swig installation to:
- not pollute the source directory
- not pollute global include_directories (use target_include_directories)
- leverage build_ext appropriately, getting rid of lost of stupid kludges
- fix ccs/unanimity versioning (bump to 2.0.1), including UNANIMITY_GIT_SHA1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -133,7 +145,7 @@ def build_extension(self, ext):
 
 setup(
     name="ConsensusCore2",
-    version="0.13.0",
```
Comment (dalexander) [inline setup.py]: This is quite nice, I was dreading having to fix this.

**Comment by dalexander** [comment]
I am going to kick the tires of this...

File: setup.py [hunk 2]
```diff
@@ -105,27 +85,32 @@ def set_generator(self, gen):
...
+        retcode = Popen(build, cwd=binaryDir).wait()
         if retcode != 0:
             raise RuntimeError("failed to build the project")
 
-
-# build the library if we haven't already
-if not os.path.exists(os.path.join(swigLib, modName)):
-    buildDir = mkdtemp()
-    try:
+class MyBuildExt(build_ext):
```
Comment (nlhepler) [inline setup.py]: I'm just  glad I finally figured out how to do the following.

**Comment by dalexander** [comment]
This seems to work pretty well--doesn't need htslib, etc. I'm going to merge this because I'm satisfied.

**Pull request merged**